src/docs/app-monitoring/sysdig-monitor-create-alert-channels.md

```diff
@@ -19,123 +19,127 @@ sort_order: 3
 ---
 
 # Create alerts and notifications in Sysdig Monitor
-Last updated: **May 31, 2023**
+Last updated: **December  18, 2023**
 
-You can create alerts based on monitoring dashboards in Sysdig Monitor, that notify your team when something needs attention.
+This document helps you create alerts based on monitoring dashboards in Sysdig Monitor that help notify your team when something needs attention.
 
-Here are some steps on how to setup the Sysdig alerts with [Rocket.Chat](https://chat.developer.gov.bc.ca/).
+Additionally, you can find steps on how to set up Sysdig alerts with [Rocket.Chat](https://chat.developer.gov.bc.ca/).
```

Comment:
@Pilargit12 this changes the meaning of this section. It should be something like 
```suggestion
These steps guide you through setting up Sysdig to post alerts via [Rocket.Chat](https://chat.developer.gov.bc.ca/).
```